modular_skyrat/modules/modular_items/lewd_items/code/lewd_arousal/pleasure.dm

```diff
@@ -1,28 +1,10 @@
-/mob/living/carbon/human/proc/get_pleasure()
-	return pleasure
-
 /mob/living/carbon/human/proc/adjustPleasure(pleas = 0)
-	if(stat != DEAD && client?.prefs?.read_preference(/datum/preference/toggle/erp))
+	if(stat >= DEAD)
+		return
+	if(client?.prefs?.read_preference(/datum/preference/toggle/erp))
 		pleasure += pleas
 		if(pleasure >= 100) // lets cum
 			climax(FALSE)
 	else
 		pleasure -= abs(pleas)
```

Comment:
Same deal here as with pain. Stuff like this shouldn't be necessary, turning your prefs for this off should just instantly reset all those values to zero, we shouldn't be doing logic to slowly take people back to 0 like this. 😔

Also same goes for the clamp as for the other two I just went past.